lib/importexport/helpers.js

```diff
@@ -123,8 +123,7 @@ var _importZip = function(filePath, cb, retries){
     if (!zipEntries || zipEntries.length ===0){
       return cb('No zip entries found');
     }
-    for (var i=0; i<zipEntries.length; i++){
-      var zipEntry = zipEntries[i];
+    zipEntries.forEach(function(zipEntry) {
```

Comment:
the problem was at https://github.com/thailekha/fh-db/blob/ecd09ede028bccf3dc05d3d69c114f6a5b937375/lib/importexport/helpers.js#L152 
a function is constructed here and it's dependent on `data` which is a pointer. If we use `for` loop then this pointer will always point the last item if zipEntries after the `for` loop finishes. `forEach` creates a new scope for every iteration I think.